v2/src/Interfaces.ts

```diff
@@ -21,3 +21,8 @@ export interface CarouselVehicle {
   terminal: string,
   delay: number
 }
+
+export interface Point {
```

Comment:
I think we should use GeoJSON conventions for representing points. They're represented as `[longitude, latitude]`.

Reply:
Fair enough, i can transform the response to match that format